R/repl.R

```diff
@@ -5,11 +5,22 @@
 #' run within the Python 'main' module, and will remain accessible after the
 #' REPL is detached.
 #' 
-#' @param banner Boolean; print a startup banner when launching the REPL?
+#' @param quiet Boolean; print a startup banner when launching the REPL? If
+#'   `FALSE`, the banner will be suppressed.
+#' @param modules A set of Python modules (packages) to be imported before
+#'   the REPL is launched. Defaults to the value of the `reticulate.repl.modules`
+#'   option.
 #' 
 #' @export
-python_repl <- function(banner = TRUE) {
- 
+python_repl <- function(
+  quiet = FALSE,
+  modules = getOption("reticulate.repl.modules", default = character()))
```

Comment:
This should just be a single `module` as the purpose of this is not actually to pre-import the module but to tell us which version of Python to use (this is a much nicer way of doing this than requiring the full path to a Python binary or the explicit call to a `use_` function). Also, this should be the first argument so people can write e.g.`python_repl("keras")`